package dev.mayanktiwari.bank.domain;

public enum Gender {
    MALE("Male"),
    FEMALE("Female"),
    OTHER("Other");

    private final String label;

    Gender(String label) {
        this.label = label;
    }

    public String label() {
        return label;
    }
}
